src/ws/service/synapse_handler.rs

```diff
@@ -18,12 +21,25 @@ use super::errors::{FriendshipsServiceError, FriendshipsServiceErrorResponse};
 
 /// Builds a room alias name from a vector of user addresses by sorting them and joining them with a "+" separator.
 ///
-/// * `user_ids` - A mut vector of users addresses as strings.
+/// * `acting_user` - The address of the acting user.
+/// * `second_user` - The address of the second user.
+/// * `synapse_url` -
 ///
-/// Returns the room alias name as a string.
-fn build_room_alias_name(mut user_ids: Vec<&str>) -> String {
-    user_ids.sort();
-    user_ids.join("+")
+/// Returns the encoded room alias name as a string, created from the sorted and joined user addresses.
+fn build_room_alias_name(acting_user: &str, second_user: &str, synapse_url: &str) -> String {
+    let act_user_parsed = format!("#{}", acting_user.to_ascii_lowercase());
```

Comment:
Can we add a comment here explaining that we had to do this as that was what has been done in Matrix client + SDK before?